sources/lists/myipms.go

```diff
@@ -1,95 +0,0 @@
-package lists
```

Comment:
This list has recent entries but the quality is low / false positive rate is high. Each entry in the list comes with an ID that tells what the IP is classified as. The possible IDs are listed [here](https://myip.ms/browse/scam) with the majority consisting of web crawlers. The only category that may be useful is ID 50: "User Submission - Hacker from this IP". However, I question the accuracy of user submissions and I find it unlikely we'll catch an active attacker this way considering how easy it is to spin up a new IP on a cloud host.